routes/passwords.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import PasswordController from '../controllers/PasswordController';
+import { passwordReset, validEmail } from '../middlewares/passwordReset';
+
+const router = express.Router();
+
+router.post('/forgotpassword', validEmail, PasswordController.forgotPassword);
```

Comment:
`/password/forgot` is more compact to verbose and redundant `/password/forgotpassword`

Reply:
done